linkedList/homework2.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once('linkedList.php');
+
+class LinkedList3 extends LinkedList 
+{
+    //made function without following hints at first
+    public function reverse() {
+        $currNode = $this->head; 
```

Comment:
you should add checking condition $this->head == null

Reply:
I added the condition you proposed & I made the function process `echo "List is empty"; `, when the condition is true.